openloader/data/create_compat_data/data/create/advancements/chromatic_compound.json

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
Does having these advancements in here do anything adverse if create isn't loaded?

Reply:
I don't think so, since I had some non-functional bits in these datapacks during testing (relying on a recipe that didn't exist) and they worked fine. Testing that now though.